spec/models/order_detail_spec.rb

```diff
@@ -475,13 +474,14 @@ def create_price_policy(params)
                order_detail: order_detail_without_price_policy,
               )
 
-        Timecop.travel(2.days.from_now) do
-          order_details.each do |order_detail|
-            order_detail.change_status!(OrderStatus.find_by_name("In Process"))
-            order_detail.change_status!(OrderStatus.find_by_name("Complete"))
-            order_detail.reload
-          end
+        current_time = Time.current
```

Comment:
This might even allow us to deal with nesting if we always use this method instead of the built-in ones that reset to actual current time. Not sure, though.